tests/page-objects/dashboard/Description.ts

```diff
@@ -19,29 +18,29 @@ export default class Description {
     }
 
     async close() {
-        await this.page.click(`${slct(DashboardPage.qa.dialogDashMeta)} .yc-dialog-btn-close__btn`);
+        await this.page.click(`${slct(DashMetaQa.Dialog)} .yc-dialog-btn-close__btn`);
         await this.closed();
     }
 
     async isViewMode() {
-        await this.page.waitForSelector(slct(COMMON_DASH_SELECTORS.DASH_META_EDIT_BTN));
+        await this.page.waitForSelector(slct(DashMetaQa.EditButton));
     }
 
     async isEditMode() {
-        await this.page.waitForSelector(slct(COMMON_DASH_SELECTORS.DASH_META_SAVE_BTN));
+        await this.page.waitForSelector(slct(DashMetaQa.SaveButton));
     }
 
     private async clickDescription() {
         await this.page.click(slct(COMMON_SELECTORS.ACTION_PANEL_DESCRIPTION_BTN));
     }
 
     private async opened() {
-        await this.page.$$(DashboardPage.selectors.dialogDashMeta);
+        await this.page.$$(`.${DashMetaQa.Dialog}`);
     }
 
     private async closed() {
         await waitForCondition(async () => {
-            const elements = await this.page.$$(DashboardPage.selectors.dialogDashMeta);
+            const elements = await this.page.$$(`.${DashMetaQa.Dialog}`);
```

Comment:
`slct` helper?

Reply:
The original usage confused me. Added `slct`